### PR TITLE
Revise the "Printing Parts" documentation

### PR DESCRIPTION
**Pull request by theacodes**
- Re-order information to better match cognitive flow.
- Note printer volume requirement.
- Emphasize the calibration print.
- Break various complex sentences into bullet lists.

### CODE REVIEW

File: doc/content/en/docs/mechanical-assembly/printing-parts/index.md [hunk 1]
```diff
@@ -9,51 +9,51 @@ description: >
...
 
-**Before printing the parts, print the calibration print**. This is [FDM-8000](https://github.com/index-machines/index/releases/latest/download/FDM-8000_corndog.stl), colloquially referred to as ["the corndog."](https://github.com/index-machines/index/releases/latest/download/FDM-8000_corndog.stl) To ensure you have your printer dialed into the tolerances we expect in our designs, this part has every feature in the printable parts with a tolerance requirement. After printing, you can check fit of fasteners and aluminum extrusion to ensure the parts you print will all fit as expected.
+## Printer requirements
 
-{{< container-image path="images/corndog.jpg" alt="Corndog" >}}
+These parts are designed to fit onto a printer with a build volume of at least 18 x 18 x 18 cm, such as the Prusa MINI.
```
Comment (atanisoft) [inline doc/content/en/docs/mechanical-assembly/printing-parts/index.md:18]: The print volume can be lowered slightly to 15x15x15. The largest part (corner legs) will actually fit on a 15x10 cm print bed if oriented correctly.

Comment (atanisoft) [reply doc/content/en/docs/mechanical-assembly/printing-parts/index.md:18]: Scratch 15x15, should be 20x15.

Comment (atanisoft) [reply doc/content/en/docs/mechanical-assembly/printing-parts/index.md:18]: or 18x15

Comment (theacodes) [reply doc/content/en/docs/mechanical-assembly/printing-parts/index.md]: Noted that it's possible to print on a smaller printed with some creativity.

**Pull request merged**